criu/files-reg.c

```diff
@@ -776,6 +776,7 @@ static struct collect_image_info remap_cinfo = {
 static int dump_ghost_file(int _fd, u32 id, const struct stat *st, dev_t phys_dev)
 {
 	struct cr_img *img;
+	int ret = 0;
```

Comment:
Usually, we call this variable as exit_code and initialize it to -1.

Reply:
I changed the variable name, but it needs an additional patch to fix `copy_file()` to return 0 on success.